<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nodeEditDialog</class>
 <widget class="QDialog" name="nodeEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Parameters</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="leftLayout">
         <item>
          <layout class="QFormLayout" name="dataLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="colourLayout"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Inputs</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
            <item>
             <widget class="QListWidget" name="inputs">
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QToolButton" name="addInput">
                <property name="text">
                 <string>Remove</string>
                </property>
                <property name="icon">
                 <iconset resource="calcIcons.qrc">
                  <normaloff>:/calc/persistantProperties/add.png</normaloff>:/calc/persistantProperties/add.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="removeInput">
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset resource="calcIcons.qrc">
                  <normaloff>:/calc/persistantProperties/remove.png</normaloff>:/calc/persistantProperties/remove.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Attributes</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
            <item>
             <widget class="QListWidget" name="attributes">
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QToolButton" name="addAttribute">
                <property name="text">
                 <string>Remove</string>
                </property>
                <property name="icon">
                 <iconset resource="calcIcons.qrc">
                  <normaloff>:/calc/persistantProperties/add.png</normaloff>:/calc/persistantProperties/add.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="removeAttribute">
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset resource="calcIcons.qrc">
                  <normaloff>:/calc/persistantProperties/remove.png</normaloff>:/calc/persistantProperties/remove.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Short Help</string>
           </property>
           <layout class="QVBoxLayout" name="shortHelpLayout">
            <property name="margin">
             <number>5</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Long Help</string>
           </property>
           <layout class="QVBoxLayout" name="longHelpLayout">
            <property name="margin">
             <number>5</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Source</string>
      </property>
      <layout class="QVBoxLayout" name="sourceLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteType">
       <property name="text">
        <string>Delete Node Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveType">
       <property name="text">
        <string>Save Node Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="calcIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>nodeEditDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
